database/redis/trigger_test.go

```diff
@@ -487,7 +487,7 @@ func TestTriggerStoring(t *testing.T) {
 			So(actual, ShouldResemble, []string{trigger.ID})
 
 			// Now update trigger
-			trigger = &triggers[1]
+			trigger = &testTriggers[1]
```

Comment:
why?

Reply:
we already it test file

Reply:
please read the [commit message](https://github.com/moira-alert/moira/pull/790/commits/b3d00e6d69a800be314e00f8b1423e1b4e25a988)

Reply:
ah, missed this line, ok